subsys/debug/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 zephyr_sources_ifdef(
   CONFIG_OPENOCD_SUPPORT
   openocd.c
-  )
+)
 
 add_subdirectory(sysview)
+add_subdirectory(ctf)
```

Comment:
Minor: wouldn't it be better to have the ifdef here instead  of in `subsys/debug/ctf/CMakeLists.txt`